### PR TITLE
Add follow count & user ID to user page

### LINKED ISSUE #192: User ID should be displayed somewhere on user's profile
**Is your feature request related to a problem? Please describe.**
If you go to someone's profile, say, /user/jellysquid3, there is no easy way to figure out that person's user ID directly from the frontend. 

**Describe the solution you'd like**
Maybe right underneath "Joined" and "Downloads", to have the user ID displayed there.

**Describe alternatives you've considered**
There are other ways to figure out their user ID, but if a mod's project ID is displayed on its sidebar, I don't see why a user shouldn't have the same.

### PR DESCRIPTION
**Pull request by wafflecoffee**
Fixes #192
image

### CODE REVIEW

File: pages/user/_id.vue [hunk 1]
```diff
@@ -141,6 +161,15 @@ export default {
         sum += mod.downloads
       }
 
+      return this.formatNumber(sum)
+    },
+    sumFollows() {
```
Comment (venashial) [inline pages/user/_id.vue:166]: I know `sumDownloads` was already a method, but I would either make `sumDownloads` & `sumFollows` a computed property or a data property. It doesn't *really* matter if you don't want to change it.

**Review by venashial — APPROVED** [review]
Nice PR!

**Comment by Mysterious-Dev** [comment]
I did not find any bugs. It's a good PR !

**Review by Mysterious-Dev — APPROVED** [review]
I did not find any bugs. It's a good PR !

**Comment by SwanX1** [comment]
I think "follows" should be changed to "followers" or "mod followers", because "follows" is usually the outgoing follows from the person.

**Comment by wafflecoffee** [comment]
Hmm, that is fair. However, "Mod followers" looks very :concern: and "Followers" implies it's per-user, not per-mod.
image

**Comment by Mysterious-Dev** [comment]
Just Follows is enough.

**Comment by falseresync** [comment]
Followers is better, it's less confusing than "Follows" and shorter than "Mod followers"

**Review by Redblueflame — APPROVED** [review]
Looks good to me!

**Pull request merged**